README.md

```diff
@@ -392,7 +434,6 @@ All the data (past and coming) are available publicly in JSON:
 * 7-11: [Devoxx Belgium](https://devoxx.be) - Antwerp (Belgium)
 * 10-11: [Volcamp](https://www.volcamp.io) - Clermont-Ferrand (France)
 * 15: [CYBERGOV](https://whitehallmedia.co.uk/future-events/) - London (UK)
-* 17-18: [DevFest Nantes](https://devfest.gdgnantes.com/en/) - Nantes (France)
```

Comment:
Why you remove it?

Reply:
Accident, most likely. The same goes for any of the other removals.